dipy/segment/threshold.py

```diff
@@ -37,3 +37,46 @@ def otsu(image, nbins=256):
     idx = np.argmax(variance12)
     threshold = bin_centers[:-1][idx]
     return threshold
+    
+    def otsu_3(image, nbins=256):
+        
+            """
+    Return two threshold values based on Otsu's method.
+    It is intended to work on a masked T1 brain image.
+    It is intended to give threshold values for CSF/WM and WM/GM. 
+    
+    Copied from scikit-image to remove dependency.
```

Comment:
There is also some stuff you can probably reuse from http://nipy.org/dipy/reference/dipy.segment.html#dipy.segment.mask.median_otsu